app/src/org/commcare/suite/model/graph/Graph.java

```diff
@@ -170,7 +170,9 @@ private void evaluateSeries(GraphData graphData, EvaluationContext context) {
                     for (Enumeration e = pointConfiguration.keys(); e.hasMoreElements();) {
                         String key = (String) e.nextElement();
                         String value = pointConfiguration.get(key).evaluate(refContext);
-                        expandedConfiguration.get(key).addElement(value);
+                        if (value != null) {
```

Comment:
It'd be cleaner to undo this change and update line 165 above to not add the key to `expandedConfiguration` in the first place if it's just an "empty" Text, but I haven't found a good way to do that (check for "emptiness").